services/ui-src/src/components/reports/ReportProvider.tsx

```diff
@@ -95,7 +96,9 @@ export const ReportProvider = ({ children }: Props) => {
   const fetchReportsByState = async (state: string) => {
     try {
       const result = await getReportsByState(state);
-      setReportsByState(result);
+      // Sort the states returned by the time they were created (Oldest To Newest)
```

Comment:
Nit: Want to say the reports are sorted, not the states. (Oldest to newest is in the function name so might not need it here).

```suggestion
      // Sort returned reports by time created
```